TOMM20.py

```diff
@@ -0,0 +1,263 @@
+from Bio import SeqIO
+from Bio.SeqUtils import gc_fraction
+import os
+
+ALPHABET_FOR_DNA = {'A', 'T', 'G', 'C', 'a', 't', 'g', 'c'}
+ALPHABET_FOR_RNA = {'A', 'U', 'G', 'C', 'a', 'u', 'g', 'c'}
+ALPHABET_FOR_PROTEIN = set('FLIMVSPTAYHQNKDECWRG')
+COMPLEMENT_ALPHABET_DNA = {'A': 'T', 'T': 'A', 'G': 'C', 'C': 'G',
+                           'a': 't', 't': 'a', 'g': 'c', 'c': 'g'}
+COMPLEMENT_ALPHABET_RNA = {'U': 'A', 'A': 'U', 'G': 'C', 'C': 'G',
+                           'u': 'a', 'a': 'u', 'g': 'c', 'c': 'g'}
+DICT_RNA_TO_PROTEIN = {
+    'UUU': 'F', 'UUC': 'F', 'UUA': 'L', 'UUG': 'L', 'CUU': 'L',
+    'CUC': 'L', 'CUA': 'L', 'CUG': 'L', 'AUU': 'I', 'AUC': 'I',
+    'AUA': 'I', 'GUU': 'V', 'GUC': 'V', 'GUA': 'V', 'GUG': 'V',
+    'UCU': 'S', 'UCC': 'S', 'UCA': 'S', 'UCG': 'S', 'AGU': 'S',
+    'AGC': 'S', 'CCU': 'P', 'CCC': 'P', 'CCA': 'P', 'CCG': 'P',
+    'ACU': 'T', 'ACC': 'T', 'ACA': 'T', 'ACG': 'T', 'GCU': 'A',
+    'GCC': 'A', 'GCA': 'A', 'GCG': 'A', 'UAU': 'Y', 'UAC': 'Y',
+    'UAA': 'stop', 'UAG': 'stop', 'UGA': 'stop', 'CAU': 'H',
+    'CAC': 'H', 'CAA': 'Q', 'CAG': 'Q', 'AAU': 'N', 'AAC': 'N',
+    'AAA': 'K', 'AAG': 'K', 'GAU': 'D', 'GAC': 'D', 'UGG': 'W',
+    'GAA': 'E', 'GAG': 'E', 'UGU': 'C', 'UGC': 'C', 'CGU': 'R',
+    'CGC': 'R', 'CGA': 'R', 'CGG': 'R', 'AGA': 'R', 'AGG': 'R',
+    'GGU': 'G', 'GGC': 'G', 'GGA': 'G', 'GGG': 'G', 'AUG': 'M'
+}
+
+DICT_MOLECULAR_MASS = {
+    'G': 75, 'A': 89, 'V': 117, 'L': 131, 'I': 131, 'P': 115,
+    'F': 165, 'Y': 181, 'W': 204, 'S': 105, 'T': 119, 'C': 121,
+    'M': 149, 'N': 132, 'Q': 146, 'D': 133, 'E': 147, 'K': 146,
+    'R': 174, 'H': 155
+}
+
+
+class BiologicalSequence(str):
```

Comment:
Абстрактные классы обычно наследуются от базового класса ABC из модуля abc. Но конкретно в этом случае, наследование от `str` выглядит вполне логичным :)
```suggestion
from abc import ABC

class BiologicalSequence(ABC):
```